.gitignore

```diff
@@ -1,3 +1,4 @@
+*~
```

Comment:
Why?

Reply:
It's a pretty standard emacs extension, but I'll remove, I didn't mean to git add that sorry

Reply:
Such ignores should be added to your global gitignore, not to project specific.